internal/resources/metal/connection/resource_schema.go

```diff
@@ -194,6 +195,10 @@ func resourceSchema(ctx context.Context) schema.Schema {
 					listplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"authorization_code": schema.StringAttribute{
+				Description: "Only used with Fabric Shared connection. Fabric uses this token to be able to give more detailed information about the Metal end of the network, when viewing resources from within Fabric.",
```

Comment:
Does this work with both `shared` and `shared_port_vlan`?  What are the customer-facing terms for `dedicated`, `shared`, and `shared_port_vlan` connections, and do we define that mapping somewhere in Equinix documentation?

Reply:
Yes, the authorization_code is present in both request responses, so we can use it for both requests. 
'dedicated', 'shared', and 'shared_port_vlan' are the customer-facing terms. I have referenced this Metal API documentation page.
https://deploy.equinix.com/developers/api/metal#tag/Interconnections/operation/createProjectInterconnection